spec/datasets/djornl/csv_node.yaml

```diff
@@ -26,10 +29,11 @@ properties:
     $ref: definitions.yaml#definitions/djornl_node/tair_curator_summary
   tair_short_description:
     $ref: definitions.yaml#definitions/djornl_node/tair_short_description
+  # comma-separated array of GO terms
   go_terms:
     type: string
     format: regex
-    pattern: "^(GO:\\d{7}, ?)*(GO:\\d{7})?$"
+    pattern: ^(GO:\d{7}, ?)*(GO:\d{7})?$
```

Comment:
If you want to still quote things (eg due to colons), you can use single quotes which dont require escaping backslashes. Here's a reference: https://docs.octoprint.org/en/master/configuration/yaml.html#scalars